.circleci/config.yml

```diff
@@ -1,23 +1,23 @@
 version: 2.1
```

Comment:
Let's remove this file from git
<img width="367" alt="CleanShot 2022-06-20 at 12 31 04@2x" src="https://user-images.githubusercontent.com/3817380/174583172-ea8cefa8-7af3-457d-84e4-cc5d2f805dc7.png">